juri/index.html

```diff
@@ -11,6 +11,61 @@
 </head>
 
 <body>
+  <header>
+   <h1>March 9, 2020</h1> 
+   5 Incomplete, 5 completed
+  </header>
+  <hr>
+  <main>
+    <section>
+     <h2>Incomplete</h2>
+     <input type="checkbox" name="upload" id="upload">
+     <label for="upload">Upload 1099-R to TurboTax</label>
+     <br>
+     &#x1F4B0 Finance
+     <br>
+     <input type="checkbox" name="submit" id="submit">
+     <label for="submit">Submit 2019 tax return</label>
+     <br>
+     &#x1F4B0 Finance
+     <br>
+     <input type="checkbox" name="print" id="print">
+     <label for="print">Print parking passes</label>
+     <br>
+     &#x1F49E Wedding
+     <br>
+     <input type="checkbox" name="sign" id="sign">
+     <label for="sign">Sign contract, send back</label>
+     <br> 
+     &#x1F5A5 Freelance
+     <br>
+     <input type="checkbox" name="hand" id="hand">
+     <label for="hand">Hand sanitizer</label>
+     <br>
```

Comment:
1. Please don't use `<br>` extensively. You don't need `<br>` since you can use CSS for the same purspose.
2. I guest this must be a list. Every list item has `<input type="checkbox" />`, possible label and `<p>`